make_BAF_igv.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python3
+import sys
+import argparse
+import vcf
+
+def calc_baf(args):
+    output_file = open(args.outputfile, 'w')
+    vcf_file = args.inputfile
+    with open(vcf_file, 'r') as vcf_input_file: 
```

Comment:
can't you do just 
`   with open(args.inputfile, 'r') as vcf_input_file: `
and delete row
`vcf_file = args.inputfile`
?

Reply:
That makes perfect sense :)